<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select the file type   :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sig_list">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
